AutoCollection/Requests.ts

```diff
@@ -67,53 +67,57 @@ class AutoCollectRequests {
             });
 	    }
     }
-
+    
     /**
-     * Tracks a request
+     * Tracks a request synchronously (doesn't wait for response 'finish' event)
      */
-    public static trackRequest(client:Client, request:http.ServerRequest, response:http.ServerResponse, properties?:{ [key: string]: string; }) {
+    public static trackRequestSync(client: Client, request: http.ServerRequest, response:http.ServerResponse, ellapsedMilliseconds?: number, properties?:{ [key: string]: string; }, error?: any) {
```

Comment:
I'm new to Node.js and this SDK and I'm not sure what's the naming convention. But in C# is recommended to add "Async" suffix to async methods, not "Sync" to those which are executed synchronously.